src/main/java/org/gatorgradle/config/GatorGradleConfig.java

```diff
@@ -194,6 +197,10 @@ public void parseHeader() {
       fastBreakBuild = file.getHeader("fastfail").asBoolean();
     }
 
+    if (file.hasHeader("username-envvar")) {
```

Comment:
Having `envvar` with those two `v`s next to each other makes this hard to read. Should we perhaps name it something else? None of the other front-matter options have dashes either, so lets try and come up with a better, more consistent name. I don't have any good ideas right now, so let's discuss?

Reply:
Maybe something to do with `identifier` or `id`? It should also indicate it is an environment variable somehow though, since it should be at least somewhat self-documenting.